src/graph/GoExecutor.cpp

```diff
@@ -625,29 +630,40 @@ void GoExecutor::onVertexProps(RpcResponse &&rpcResp) {
     UNUSED(rpcResp);
 }
 
-std::vector<VertexID> GoExecutor::getDstIdsFromResps(std::vector<RpcResponse>::iterator begin,
-                                                     std::vector<RpcResponse>::iterator end) const {
-    std::unordered_set<VertexID> set;
-    for (auto it = begin; it != end; ++it) {
-        for (const auto &resp : it->responses()) {
-            auto *vertices = resp.get_vertices();
-            if (vertices == nullptr) {
-                continue;
-            }
 
-            for (const auto &vdata : *vertices) {
-                for (const auto &edata : vdata.edge_data) {
-                    for (const auto& edge : edata.get_edges()) {
-                        auto dst = edge.get_dst();
-                        if (!isFinalStep() && backTracker_ != nullptr) {
-                            backTracker_->add(vdata.get_vertex_id(), dst);
-                        }
-                        set.emplace(dst);
+void GoExecutor::getDstIdsFromResp(const RpcResponse &rpcResp,
```

Comment:
How about modifying the function name to `getUniqueDstId`

Reply:
Good idea.